### PR TITLE
Fix Coveralls submission with HTTP 0.8

### PR DESCRIPTION
**Pull request by ararslan**

### CODE REVIEW

**Comment by ararslan** [comment]
Hm, some kind of version resolution issue; we need HTTP 0.8 but something is keeping us from installing it.

File: src/coveralls.jl [hunk 1]
```diff
@@ -94,10 +94,7 @@ module Coveralls
             println("Submitting data to Coveralls...")
         end
 
-        url = "https://coveralls.io/api/v1/jobs"
-        body = HTTP.Form(makebody(data))
-        headers = ["Content-Type" => "multipart/form-data; boundary=$(body.boundary)"]
-        req = HTTP.post(url, headers, body)
+        req = HTTP.post("https://coveralls.io/api/v1/jobs", HTTP.Form(makebody(data)))
```
Comment (vtjnash) [inline src/coveralls.jl]: missing an identical update later in the file

Comment (ararslan) [reply src/coveralls.jl]: Good catch, thanks!

File: REQUIRE [hunk 2]
```diff
@@ -1,4 +1,4 @@
 julia 1.0
```
Comment (HarrisonGrodin) [inline REQUIRE]: Afaict, Coveralls still fails on 0.7, I assume due to a versioning issue. Should this be changed to `0.7`?

**Pull request merged**